dev/com.ibm.ws.jaxrs.2.0.server/src/com/ibm/ws/jaxrs20/server/LibertyJaxRsServerFactoryBean.java

```diff
@@ -69,7 +69,8 @@
     private final static TraceComponent tc = Tr.register(LibertyJaxRsServerFactoryBean.class);
 
     private final List<JaxRsFactoryBeanCustomizer> beanCustomizers = new LinkedList<JaxRsFactoryBeanCustomizer>();
-    private final Map<String, Object> beanCustomizerContexts = new HashMap<String, Object>();
+//    private final Map<String, Object> beanCustomizerContexts = new HashMap<String, Object>();
```

Comment:
and this comment: `//    private final Map<String, Object> beanCustomizerContexts = new HashMap<String, Object>();`